### DISCUSSION TITLE
Generating a synthetic model

### DISCUSSION THREAD

**Discussion by Esh15** [fatiando/community#82] [Q&A] [answered]
Hi, how do I generate a synthetic model? I noticed that fatiando is not recognised on python 3.

**Comment by santisoler** [comment, marked as answer]
Hi @Esh15! Thanks for opening this issue.

You're right, the old `fatiando` Python 2.7 package has been deprecated. We have moved to a new set of tools instead, like Harmonica.

In Harmonica we have functions to forward model the gravity fields of point sources, prisms and tesseroids. You build your synthetic model using these geometries and using these functions to compute their fields. Is this your goal?

You can checkout our User Guide for detailed description of how to run [Forward Models](https://www.fatiando.org/harmonica/latest/user_guide/forward_modelling/index.html) using Harmonica.


---

I'm converting this Issue into a Discussion inside our forum. @Esh15 you should be able to comment on this after the move. If you experience any problem, please let us know.

**Reply by Esh15** [reply]
Thank you, I am trying to do a forward modelling on a magnetic data. Can harmonica do this?

**Reply by santisoler** [reply]
Sadly not at the moment. We have just made an early release of [Choclo](https://www.fatiando.org/choclo) that will allow us to forward model magnetic fields within Harmonica.

For the moment you could use the forward modelling capabilities of SimPEG: https://docs.simpeg.xyz/

**Comment by Esh15** [comment]
Thank you for your time. This was helpful
